### PR TITLE
Deployable Barrier Buff v1

### PR DESCRIPTION
**Pull request by Super3222**
### Intent of your Pull Request

Currently deployable barriers are the easiest thing to get through, even without an emag.

Currently you could destroy a barrier with a toolbox or a fire extinguisher under 10 seconds and it goes completely unnoticed until someone walks over and sees the rods left over from the barrier and the small environment damage.

So my mission is to make it loud enough for someone from a long, but fair distance to hear it.

I could use feedback from people who play security often.

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
Make it explode and damage the person who broke it

**Comment by Time-Green** [comment]
Make it send a message on security channel where and who destroyed it.

**Comment by EMT321** [comment]
Making them explode seems like a beneficial way to stop people smacking em open.

Currently, most people dont even try.

**Comment by DavidEdwards** [comment]
This PR is small enough to be put in without too much fuss for balance. Lets see how it goes.

File: html/changelogs/Super3222-deployablebuff.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Your name.  
+author: N3X15
```
Comment (Cruix) [inline html/changelogs/Super3222-deployablebuff.yml:2]: Might want to update this.

**Pull request merged**